model-engine/model_engine_server/inference/batch_inference/vllm_batch.py

```diff
@@ -265,7 +270,9 @@ def tool_func(text: str, past_context: Optional[str]):
                 or gen_item.remaining_tokens <= 0
             ):
                 gen_item.completed = True
-                continue
+
+        pool = ThreadPool(CPU_COUNT)
```

Comment:
it's probably fine to use single process to start subprocesses when CPU number is low